tests/install_vm.py

```diff
@@ -112,19 +127,26 @@ def main():
     # The kernel option 'net.ifnames=0' is used to disable predictable network
     # interface names, for more details see:
     # https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/
-    command = 'virt-install --name={domain} --memory={ram} --vcpus={cpu} --os-variant={variant} --hvm --accelerate --network=default --disk path={disk_path},size=20,format=qcow2 --initrd-inject={kickstart} --extra-args="inst.ks=file:/{ks_basename} ksdevice=eth0 net.ifnames=0" --wait=-1 --vnc --location={url}'.format(**data.__dict__)
+    if data.libvirt == "qemu:///system":
+        command = 'virt-install --connect={libvirt} --name={domain} --memory={ram} --vcpus={cpu} --os-variant={variant} --hvm --accelerate --network=default --disk path={disk_path},size=20,format=qcow2 --initrd-inject={kickstart} --extra-args="inst.ks=file:/{ks_basename} ksdevice=eth0 net.ifnames=0" --graphics=none --noautoconsole --wait=-1 --location={url}'.format(**data.__dict__)
+    else:
+        command = 'virt-install --connect={libvirt} --name={domain} --memory={ram} --vcpus={cpu} --os-variant={variant} --hvm --accelerate --network bridge=virbr0 --disk path={disk_path},size=20,format=qcow2 --initrd-inject={kickstart} --extra-args="inst.ks=file:/{ks_basename} ksdevice=eth0 net.ifnames=0" --graphics=none --noautoconsole --wait=-1 --location={url}'.format(**data.__dict__)
     if data.dry:
         print("\nThe following command would be used for the VM installation:")
         print(command)
     else:
         os.system(command)
 
     print("\nTo determine the IP address of the {0} VM use:".format(data.domain))
-    print("sudo virsh domifaddr {0}\n".format(data.domain))
+    if data.libvirt == "qemu:///system":
+        print("sudo virsh domifaddr {0}\n".format(data.domain))
+    else:
+        print("arp -n | grep virbr0 | grep ether\n")
```

Comment:
This can get a bit tricky, if you have a lot of machines:
```
$ arp -n | grep virbr0 | grep ether
192.168.122.22           ether   52:54:00:e3:1e:da   C                     virbr0
192.168.122.32           ether   52:54:00:32:62:a0   C                     virbr0
192.168.122.219          ether   52:54:00:a0:8a:4a   C                     virbr0
192.168.122.177          ether   52:54:00:e0:2c:00   C                     virbr0
192.168.122.247          ether   52:54:00:55:3a:e8   C                     virbr0
192.168.122.31           ether   52:54:00:32:62:a0   C                     virbr0
192.168.122.229          ether   52:54:00:31:fc:c0   C                     virbr0
192.168.122.218          ether   52:54:00:a0:8a:4a   C                     virbr0
192.168.122.113          ether   52:54:00:fd:ea:be   C                     virbr0
192.168.122.230          ether   52:54:00:31:fc:c0   C                     virbr0
192.168.122.176          ether   52:54:00:e0:2c:00   C                     virbr0
192.168.122.21           ether   52:54:00:e3:1e:da   C                     virbr0
192.168.122.84           ether   52:54:00:f8:37:6c   C                     virbr0
```
With `virsh domiflist <domain_name>`, you can get the mac address, and with that the ip of your machine.